CHANGES.md

```diff
@@ -11,6 +11,12 @@
   * Fixed border color of windows with alpha channel. Now all windows have the
     same opaque border color.
 
+  * Change the main loop to try to avoid [GHC bug 21708] on systems running
+    GHC 9.0 or later. A fixed GHC is forthcoming, as the bug is a blocker for
+    GHC 9.4.1.
+
+[GHC bug 21708]: https://gitlab.haskell.org/ghc/ghc/-/issues/21708
```

Comment:
Afaik the bug is specifically with 9.2.x (I can't find anyone being able to reproduce it with 9.0.x).  At this point, we might also mention that it's fixed with [9.2.4](https://discourse.haskell.org/t/ghc-9-2-4-released/4851)